plugins/jobs/src/js/components/JobsBreadcrumbs.d.ts

```diff
@@ -0,0 +1,10 @@
+import { Component } from "react";
+import JobTree from "#SRC/js/structs/JobTree";
+import Job from "#SRC/js/structs/Job";
+
+export default class JobsBreadcrumbs extends Component {
+  props: {
```

Comment:
I thought we were going with interfaces for props :)

Reply:
yeah, missed that… I'd like to fix it in a second PR, this one is green and I'd like to have it merged asap 👍